v1.1/admin_ui_databases_page.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Database Page
+toc: false
+feedback: false
+---
+
+The Databases page of the Admin UI provides details of the databases configured, the tables in each database, and the grants assigned to each user.
+
+<div id="toc"></div>
+
+## Tables View  
+
+To view the Tables view, [access the Admin UI](explore-the-admin-ui.html#access-the-admin-ui) and select **Databases** from the left-hand navigation bar. The **Tables** view is displayed.
+
+<img src="{{ 'images/admin_ui_database_tables_view.png' | relative_url }}" alt="CockroachDB Admin UI Database Tables View" style="border:1px solid #eee;max-width:100%" />
+
+The Tables view shows details of the system table as well as the tables in your databases. The following details are displayed for each table:
+
+Metric | Description
+--------|----
+Table Name | The name of the table.
+Size | The size of the table in iB.
+Ranges | The number of ranges in the table.
+`# of Columns | The number of columns in the table.
```

Comment:
weird backticks

Reply:
Markdown convention, else # indicates Heading 1

Reply:
It's coming through in the preview, though?
![screen shot 2017-09-27 at 9 53 25 am](https://user-images.githubusercontent.com/793969/30917314-ba5866ea-a369-11e7-9ef7-5f55cbf58e9e.png)

Reply:
Oh yeah..wrong escape character. Fixed it. Thanks for pointing it out :)